### PR TITLE
Update input files for CBGCv2 land-atm 1850 and 20tr compsets

### PR DESCRIPTION
**Pull request by BunnyVon**
add elm input files to the land-atm 1850 and 20tr compsets and make sure the simulations use correct namelist files. 

[NBFB]

### CODE REVIEW

File: components/elm/bld/namelist_files/namelist_defaults.xml [hunk 1]
```diff
@@ -562,7 +562,7 @@ this mask will have smb calculated over the entire global land surface
 <flndtopo hgrid="0.9x1.25"  >lnd/clm2/griddata/topodata_0.9x1.25_USGS_070110.nc</flndtopo>
 <flndtopo hgrid="1.9x2.5"   >lnd/clm2/griddata/topodata_1.9x2.5_USGS_061130.nc</flndtopo>
 <flndtopo hgrid="48x96"     >lnd/clm2/griddata/topodata_48x96_USGS_070110.nc</flndtopo>
-<flndtopo hgrid="r05"       >lnd/clm2/surfdata_map/surfdata_0.5x0.5_simyr1850_c200609.nc</flndtopo>
+<flndtopo hgrid="r05"       >/lnd/clm2/surfdata_map/surfdata_0.5x0.5_simyr1850_c211019.nc</flndtopo>
```
Comment (jonbob) [inline components/elm/bld/namelist_files/namelist_defaults.xml:565]: @BunnyVon - I don't think you want the leading "/" here

File: components/elm/cime_config/config_component.xml [hunk 2]
```diff
@@ -68,10 +68,13 @@
       <value compset="PDAY.*_ELM"            >1850-2100_rcp4.5_transient</value> 
       <!-- CMIP6 DECK compsets and related BGC compsets -->
       <value compset="1850(?:SOI)?_EAM%CMIP6_ELM">1850_CMIP6_control</value>
+      <value compset="1850(?:SOI)?_EAM%CMIP6_ELM.*_BGC%LNDATM">1850_CMIP6bgc_control</value>
+      <value compset="1850(?:SOI)?_EAM%CMIP6_ELM.*_BGC%BCRC">1850_CMIP6bgc_control</value>
       <value compset="1850S_EAM%CMIP6_ELM">1850_SCMIP6_control</value>
       <value compset="1850(?:SOI)?_EAM%CMIP6_ELM*_BGC%BCRC">1850_CMIP6bgc_control</value>
       <value compset="1850S_EAM%CMIP6_ELM">1850_SCMIP6_control</value>
       <value compset="20TR.*_EAM%.*CMIP6.*_ELM">20thC_CMIP6_transient</value>
+      <value compset="20TR_EAM%CMIP6_ELM.*_BGC%BLNDATM">20thC_CMIP6bgc_transient</value>
```
Comment (jonbob) [inline components/elm/cime_config/config_component.xml:77]: should this be "BGC%BLNDATM" or just "BGC%LNDATM"?

**Review by bishtgautam — CHANGES_REQUESTED** [review]

File: components/elm/bld/namelist_files/namelist_defaults.xml [hunk 3]
```diff
@@ -115,10 +115,10 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile                 >lnd/clm2/paramdata/clm_params_c180301.nc</paramfile>
+<paramfile                 >lnd/clm2/paramdata/clm_params_c180524_phs.nc</paramfile>
```
[the remark this answers is not in the document]
Comment (jenniferholm) [reply components/elm/bld/namelist_files/namelist_defaults.xml:118]: Sounds good to me to remove the "phs"

File: components/elm/cime_config/config_component.xml (hunk 2 above)
Comment (BunnyVon) [reply components/elm/cime_config/config_component.xml:77]: just "BGC%LNDATM

File: components/elm/cime_config/config_component.xml [hunk 4]
```diff
@@ -68,10 +68,13 @@
       <value compset="PDAY.*_ELM"            >1850-2100_rcp4.5_transient</value> 
       <!-- CMIP6 DECK compsets and related BGC compsets -->
       <value compset="1850(?:SOI)?_EAM%CMIP6_ELM">1850_CMIP6_control</value>
+      <value compset="1850(?:SOI)?_EAM%CMIP6_ELM.*_BGC%LNDATM">1850_CMIP6bgc_control</value>
+      <value compset="1850(?:SOI)?_EAM%CMIP6_ELM.*_BGC%BCRC">1850_CMIP6bgc_control</value>
       <value compset="1850S_EAM%CMIP6_ELM">1850_SCMIP6_control</value>
       <value compset="1850(?:SOI)?_EAM%CMIP6_ELM*_BGC%BCRC">1850_CMIP6bgc_control</value>
```
[the remark this answers is not in the document]
Comment (BunnyVon) [reply components/elm/cime_config/config_component.xml:74]: fixed.

File: components/elm/bld/namelist_files/namelist_defaults.xml (hunk 1 above)
Comment (BunnyVon) [reply components/elm/bld/namelist_files/namelist_defaults.xml:565]: fixed.

File: components/elm/bld/namelist_files/namelist_defaults.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (BunnyVon) [reply components/elm/bld/namelist_files/namelist_defaults.xml:118]: removed

**Comment by rljacob** [comment]
@BunnyVon please change the PR title.   Remove the "Create PR" part and just say "add".

File: components/elm/bld/namelist_files/namelist_defaults.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (bishtgautam) [reply components/elm/bld/namelist_files/namelist_defaults.xml]: I realize that my last comment wasn't clear. If the `clm_params_c180524_phs.nc` is the latest parameter file (with updated or new parameters), then the filename should reflect that it was created recently. e.g. `clm_params_c220908.nc` assuming it was created on 09-08-2022.

[the remark this answers is not in the document]
Comment (BunnyVon) [reply components/elm/bld/namelist_files/namelist_defaults.xml]: @jenniferholm what do you think about @bishtgautam's comment above?

**Review by jonbob — APPROVED** [review]
Approved, based on visual inspection and some testing

File: components/elm/bld/namelist_files/namelist_defaults.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jenniferholm) [reply components/elm/bld/namelist_files/namelist_defaults.xml]: We'll need to confirm with Yilin on the date that she made this file.  It was before 2022, and the "180524" might in fact be the created date.

[the remark this answers is not in the document]
Comment (yfang-repo) [reply components/elm/bld/namelist_files/namelist_defaults.xml]: clm_params_c180524_phs.nc was created on Nov 24, 2021, to be consistent with the 180524 version, the ELMv1 CBGC default for CTC. I just copied to clm_params_c211124.nc.

[the remark this answers is not in the document]
Comment (BunnyVon) [reply components/elm/bld/namelist_files/namelist_defaults.xml]: @yfang-repo thank you. @bishtgautam I have this line fixed and the link `clm_params_c211124.nc` to `clm_params_c180524_phs.nc` in the inputdata directory on cori.

[the remark this answers is not in the document]
Comment (bishtgautam) [reply components/elm/bld/namelist_files/namelist_defaults.xml]: @BunnyVon On Cori, please unlink `clm_params_c211124.nc` and copy the file from Compy.

FYI. I have updated the file on E3SM server https://web.lcrc.anl.gov/public/e3sm/inputdata/lnd/clm2/paramdata/.

[the remark this answers is not in the document]
Comment (BunnyVon) [reply components/elm/bld/namelist_files/namelist_defaults.xml]: done @bishtgautam

**Comment by bpbond** [comment]
Confirming that this is the last PR before issuing v2.1alpha tag.

**Review by bishtgautam — APPROVED** [review]

**Comment by njeffery** [comment]
@BunnyVon :  how many remaining reviews do you you need before this PR can be merged?  @jenniferholm  is out all this week and won't be available.  @bpbond would like to see this merged asap if possible.

**Comment by rljacob** [comment]
@peterdschwartz please start merging this.

**Comment by peterdschwartz** [comment]
@BunnyVon The test `SMS.r05_r05.I1850ELMCN.chrysalis_intel.elm-qian_1948` seems to be failing due to an inconsistency in `create_crop_landunit = .false.` being used with the new surface data sets.
Error message:
```
6:  ENDRUN:
  6:   ERROR: New format surface datasets require create_crop_landunit TRUEERROR in s
  6:  urfrdMod.F90 at line 1109
```

**Comment by peterdschwartz** [comment]
Made further changes so that create_crop_landunit is properly set for the new surface data files.   Tested after merging to next didn't have any tests fail and a lot of DIFFs that are expected.  So I will plan to push it to next on Monday

**Comment by peterdschwartz** [comment]
Merged to next

**Comment by peterdschwartz** [comment]
I reverted the merge due to all the DIFFs in the production test suite that are caused by the new clm parameter file.  Has it been verified that the use of this parameter file doesn't result in a different climate for these runs?  If it does, then the setting of the new parameter file should likely be done only if `use_cn = .true.` like with the new surface data set and flanduse_timeseries.  

The new commit adjusting the setting of `flanduse_timeseries` had to be done due to a FAIL in r05 runs with `use_cn=.false.`

**Comment by jenniferholm** [comment]
Hi Peter - it's my understanding that the new parameter file will not cause different climate.  The updates/changes to this new parameter file should only impact plant hydraulic features, so only change results when “use_hydrstress = .true.”. But this parameter file can also be used for all other ELM runs, even when “use_hydrstress = .false.”
I think you are right that the new parameter file should only be used if use_cn = .true. But I'd like to have other double confirm this.  
I hope this helps answer your question.

**Comment by rljacob** [comment]
Is @BunnyVon going to add the commit to restrict when the param file is used?  Once that is in, we can remerge this to next.

**Comment by BunnyVon** [comment]
> use_hydrstress

**Comment by BunnyVon** [comment]
> Hi Peter - it's my understanding that the new parameter file will not cause different climate. The updates/changes to this new parameter file should only impact plant hydraulic features, so only change results when “use_hydrstress = .true.”. But this parameter file can also be used for all other ELM runs, even when “use_hydrstress = .false.” I think you are right that the new parameter file should only be used if use_cn = .true. But I'd like to have other double confirm this. I hope this helps answer your question.

@jenniferholm I have checked my lnd_in for the test run using this branch. I can't locate `use_hydrstress`

**Comment by BunnyVon** [comment]
> Is @BunnyVon going to add the commit to restrict when the param file is used? Once that is in, we can remerge this to next.

@jonbob I think @bishtgautam already pushed the commit to restrict when the param file is used. @bishtgautam could you confirm?

**Comment by peterdschwartz** [comment]
I can make the change for the parameter file.

**Comment by rljacob** [comment]
telecon notes:  some namelist variable issues still to resolve.

**Comment by BunnyVon** [comment]
> telecon notes: some namelist variable issues still to resolve.

@rljacob could you elaborate more? Based on my last communication last Friday (9/30), the variable list is final and ready for production runs.

**Comment by peterdschwartz** [comment]
@BunnyVon My fix of just checking "use_cn" worked for the production run test suite but was insufficient for FATES test  with ECA in particular since they also have `use_cn = .false. `

I have added more checks for use_fates in setting the paramfile, which looking at the Namelist comparisons seem to be ok now but I am waiting for test suite runs to say for sure that it works properly for everything.  I will be making another commit to update these changes.

**Comment by peterdschwartz** [comment]
I'll try merging this again with the latest changes.  Here were the tests that had DIFFs or NLFAILs (some unrelated to this PR) from the e3sm_developer test. Including the list in case there's any obvious tests that are wrongly affected.
```
  ERS.f09_g16.I1850ELMCN.chrysalis_intel (Overall: DIFF) details:
  ERS.f09_g16.I1850ELMCN.chrysalis_intel.elm-bgcinterface (Overall: DIFF) details:
  ERS.f19_f19.I1850ELMCN.chrysalis_intel (Overall: DIFF) details:
  ERS.f19_f19.I20TRELMCN.chrysalis_intel (Overall: DIFF) details:
  ERS.f19_g16.I1850CNECACNTBC.chrysalis_intel.elm-eca (Overall: DIFF) details:
  ERS.f19_g16.I1850CNECACTCBC.chrysalis_intel.elm-eca (Overall: DIFF) details:
  ERS.f19_g16.I1850ELM.chrysalis_intel.elm-betr (Overall: DIFF) details:
  ERS.f19_g16.I1850GSWCNPECACNTBC.chrysalis_intel.elm-eca_f19_g16_I1850GSWCNPECACNTBC (Overall: DIFF) details:
  ERS.f19_g16.I1850GSWCNPRDCTCBC.chrysalis_intel.elm-ctc_f19_g16_I1850GSWCNPRDCTCBC (Overall: DIFF) details:
  ERS.f19_g16.I20TRGSWCNPECACNTBC.chrysalis_intel.elm-eca_f19_g16_I20TRGSWCNPECACNTBC (Overall: DIFF) details:
  ERS.f19_g16.I20TRGSWCNPRDCTCBC.chrysalis_intel.elm-ctc_f19_g16_I20TRGSWCNPRDCTCBC (Overall: DIFF) details:
  ERS_Ld20.f45_f45.IELMFATES.chrysalis_intel.elm-fates (Overall: DIFF) details:
  ERS.ne30pg2_r05_EC30to60E2r2.GPMPAS-JRA.chrysalis_intel.mosart-rof_ocn_2way (Overall: DIFF) details:
  ERS.r05_r05.IELM.chrysalis_intel.elm-V2_ELM_MOSART_features (Overall: DIFF) details:
  SMS_Ld1.hcru_hcru.I1850CRUELMCN.chrysalis_intel (Overall: DIFF) details:
  SMS_Ld20.f45_f45.IELMFATES.chrysalis_intel.elm-fates_rd (Overall: DIFF) details:
  SMS_Ld30.f45_f45.IELMFATES.chrysalis_intel.elm-fates_satphen (Overall: DIFF) details:
  SMS_Ly2_P1x1.1x1_smallvilleIA.IELMCNCROP.chrysalis_intel.elm-force_netcdf_pio (Overall: DIFF) details:
  SMS_Ly2_P1x1.1x1_smallvilleIA.IELMCNCROP.chrysalis_intel.elm-lulcc_sville (Overall: DIFF) details:
  SMS_Ly2_P1x1.1x1_smallvilleIA.IELMCNCROP.chrysalis_intel.elm-per_crop (Overall: DIFF) details:
  SMS.r05_r05.I1850ELMCN.chrysalis_intel.elm-qian_1948 (Overall: DIFF) details:

  ERIO.ne30_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
  ERS.f09_g16.I1850GSWCNPRDCTCBC.chrysalis_intel.elm-vstrd (Overall: NLFAIL) details:
  ERS.f19_g16.I1850CNRDCTCBC.chrysalis_intel.elm-rd (Overall: NLFAIL) details:
  ERS.f19_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
  ERS.hcru_hcru.I20TRGSWCNPRDCTCBC.chrysalis_intel.elm-erosion (Overall: NLFAIL) details:
  ERS.ne30_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
  NCK.f19_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
  SMS.ne30_f19_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details
```

**Comment by BunnyVon** [comment]
> I'll try merging this again with the latest changes. Here were the tests that had DIFFs or NLFAILs (some unrelated to this PR) from the e3sm_developer test. Including the list in case there's any obvious tests that are wrongly affected.
> 
> ```
>   ERS.f09_g16.I1850ELMCN.chrysalis_intel (Overall: DIFF) details:
>   ERS.f09_g16.I1850ELMCN.chrysalis_intel.elm-bgcinterface (Overall: DIFF) details:
>   ERS.f19_f19.I1850ELMCN.chrysalis_intel (Overall: DIFF) details:
>   ERS.f19_f19.I20TRELMCN.chrysalis_intel (Overall: DIFF) details:
>   ERS.f19_g16.I1850CNECACNTBC.chrysalis_intel.elm-eca (Overall: DIFF) details:
>   ERS.f19_g16.I1850CNECACTCBC.chrysalis_intel.elm-eca (Overall: DIFF) details:
>   ERS.f19_g16.I1850ELM.chrysalis_intel.elm-betr (Overall: DIFF) details:
>   ERS.f19_g16.I1850GSWCNPECACNTBC.chrysalis_intel.elm-eca_f19_g16_I1850GSWCNPECACNTBC (Overall: DIFF) details:
>   ERS.f19_g16.I1850GSWCNPRDCTCBC.chrysalis_intel.elm-ctc_f19_g16_I1850GSWCNPRDCTCBC (Overall: DIFF) details:
>   ERS.f19_g16.I20TRGSWCNPECACNTBC.chrysalis_intel.elm-eca_f19_g16_I20TRGSWCNPECACNTBC (Overall: DIFF) details:
>   ERS.f19_g16.I20TRGSWCNPRDCTCBC.chrysalis_intel.elm-ctc_f19_g16_I20TRGSWCNPRDCTCBC (Overall: DIFF) details:
>   ERS_Ld20.f45_f45.IELMFATES.chrysalis_intel.elm-fates (Overall: DIFF) details:
>   ERS.ne30pg2_r05_EC30to60E2r2.GPMPAS-JRA.chrysalis_intel.mosart-rof_ocn_2way (Overall: DIFF) details:
>   ERS.r05_r05.IELM.chrysalis_intel.elm-V2_ELM_MOSART_features (Overall: DIFF) details:
>   SMS_Ld1.hcru_hcru.I1850CRUELMCN.chrysalis_intel (Overall: DIFF) details:
>   SMS_Ld20.f45_f45.IELMFATES.chrysalis_intel.elm-fates_rd (Overall: DIFF) details:
>   SMS_Ld30.f45_f45.IELMFATES.chrysalis_intel.elm-fates_satphen (Overall: DIFF) details:
>   SMS_Ly2_P1x1.1x1_smallvilleIA.IELMCNCROP.chrysalis_intel.elm-force_netcdf_pio (Overall: DIFF) details:
>   SMS_Ly2_P1x1.1x1_smallvilleIA.IELMCNCROP.chrysalis_intel.elm-lulcc_sville (Overall: DIFF) details:
>   SMS_Ly2_P1x1.1x1_smallvilleIA.IELMCNCROP.chrysalis_intel.elm-per_crop (Overall: DIFF) details:
>   SMS.r05_r05.I1850ELMCN.chrysalis_intel.elm-qian_1948 (Overall: DIFF) details:
> 
>   ERIO.ne30_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
>   ERS.f09_g16.I1850GSWCNPRDCTCBC.chrysalis_intel.elm-vstrd (Overall: NLFAIL) details:
>   ERS.f19_g16.I1850CNRDCTCBC.chrysalis_intel.elm-rd (Overall: NLFAIL) details:
>   ERS.f19_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
>   ERS.hcru_hcru.I20TRGSWCNPRDCTCBC.chrysalis_intel.elm-erosion (Overall: NLFAIL) details:
>   ERS.ne30_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
>   NCK.f19_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details:
>   SMS.ne30_f19_g16_rx1.A.chrysalis_intel (Overall: NLFAIL) details
> ```

@jenniferholm this is a list of the tests that are changed by this PR, most of which are land simulations. Do you see if anything is obviously wrong? Thank you~





New
[8:35](https://acmeclimate.slack.com/archives/D043JSKPQ7K/p1665588913331249)
If it looks alright, then I'll alert Rob that it's ready to try merging again.

**Comment by BunnyVon** [comment]
@jenniferholm  this is a list of the tests that are changed by this PR, most of which are land simulations. Do you see if anything is obviously wrong? If it looks alright, then Peter will alert Rob that it's ready to try merging again.

**Comment by jenniferholm** [comment]
I have to admit, I'm not the best at reviewing and looking over test outputs (of DIFFs and NLFAILs).  But I don't think anything looks glaring wrong from the test outputs, especially since those are not the lnd-atm compsets or tests. 

The one thing I am wondering about in the latest commit, is why was the "nu_com=RD" case removed? This is the default CNP case for land simulations, and should not be removed.

**Comment by peterdschwartz** [comment]
@jenniferholm To clarify, the `nu_com = 'RD'` wasn't removed as a case -- it is still the default value for nu_com in the source code (elm_varctl.F90) and the namelist. This caused issues where the ELMBuildNamelist.pm script wouldn't match any `paramfile` due to some configurations leaving out `nu_com` entirely from lnd_in as it is the hard-coded default value.

Since it is the default value, the simplest solution was to remove the check as only 'ECA' results in a different `paramfile` as far as I could tell.  A different solution seemed that it would require untangling the order in which functions are executed in ELMBuildNamelist.pm and why exactly why nu_com gets skipped sometimes.

For the LNDATM tests done in the full `e3sm_integration` suite I just looked at the nml list diffs :
```
SMS_Ln5.ne30pg2_r05_EC30to60E2r2.BGCEXP_LNDATM_CNPRDCTC_1850.chrysalis_intel
Differences in namelist 'seq_maps.rc':
  BASE: atm2rof_fmapname = 'idmap_ignore'
  COMP: atm2rof_fmapname = '/lcrc/group/e3sm/data/inputdata/cpl/gridmaps/ne30pg2/map_ne30pg2_to_r05_mono.200220.nc'
  BASE: atm2rof_smapname = 'idmap_ignore'
  COMP: atm2rof_smapname = '/lcrc/group/e3sm/data/inputdata/cpl/gridmaps/ne30pg2/map_ne30pg2_to_r05_mono.200220.nc'
  BASE: ocn2rof_smapname = 'idmap_ignore'
  COMP: ocn2rof_smapname = '/lcrc/group/e3sm/data/inputdata/cpl/cpl6/map_EC30to60E2r2_to_r05_neareststod.220728.nc'
  BASE: rof2ocn_ice_rmapname = 'idmap'
  COMP: rof2ocn_ice_rmapname = '/lcrc/group/e3sm/data/inputdata/cpl/cpl6/map_r05_to_EC30to60E2r2_smoothed.r150e300.201005.nc'
  BASE: rof2ocn_liq_rmapname = 'idmap'
  COMP: rof2ocn_liq_rmapname = '/lcrc/group/e3sm/data/inputdata/cpl/cpl6/map_r05_to_EC30to60E2r2_smoothed.r150e300.201005.nc'

Differences in namelist 'elm_inparm':
  BASE: co2_type = 'diagnostic'
  COMP: co2_type = 'prognostic'
  BASE: create_crop_landunit = .false.
  COMP: create_crop_landunit = .true.
  BASE: fsurdat = surfdata_0.5x0.5_simyr1850_c200609.nc'
  COMP: fsurdat = surfdata_0.5x0.5_simyr1850_c211019.nc'
  BASE: paramfile = clm_params_c180524.nc'
  COMP: paramfile = clm_params_c211124.nc'
  BASE: stream_fldfilename_ndep = fndep_clm_hist_simyr1849-2006_1.9x2.5_c100428.nc'
  COMP: stream_fldfilename_ndep = fndep_elm_cbgc_exp_simyr1849-2101_1.9x2.5_c190103.nc'
  BASE: stream_fldfilename_popdens = clmforc.Li_2012_hdm_0.5x0.5_AVHRR_simyr1850-2010_c130401.nc'
  COMP: stream_fldfilename_popdens = elmforc.ssp5_hdm_0.5x0.5_simyr1850-2100_c190109.nc'
Differences in namelist 'dynamic_subgrid':
  found extra variable: 'do_harvest'
  found extra variable: 'do_transient_pfts'
  found extra variable: 'flanduse_timeseries'
  BASE: lnd_rof_coupling_nstep = 1
  COMP: lnd_rof_coupling_nstep = 6
```

```
SMS_Ln5.ne30pg2_r05_EC30to60E2r2.BGCEXP_LNDATM_CNPRDCTC_20TR.chrysalis_intel
Differences in namelist 'seq_maps.rc':
  BASE: atm2rof_fmapname = 'idmap_ignore'
  COMP: atm2rof_fmapname = '/lcrc/group/e3sm/data/inputdata/cpl/gridmaps/ne30pg2/map_ne30pg2_to_r05_mono.200220.nc'
  BASE: atm2rof_smapname = 'idmap_ignore'
  COMP: atm2rof_smapname = '/lcrc/group/e3sm/data/inputdata/cpl/gridmaps/ne30pg2/map_ne30pg2_to_r05_mono.200220.nc'
  BASE: ocn2rof_smapname = 'idmap_ignore'
  COMP: ocn2rof_smapname = '/lcrc/group/e3sm/data/inputdata/cpl/cpl6/map_EC30to60E2r2_to_r05_neareststod.220728.nc'
  BASE: rof2ocn_ice_rmapname = 'idmap'
  COMP: rof2ocn_ice_rmapname = '/lcrc/group/e3sm/data/inputdata/cpl/cpl6/map_r05_to_EC30to60E2r2_smoothed.r150e300.201005.nc'
  BASE: rof2ocn_liq_rmapname = 'idmap'
  COMP: rof2ocn_liq_rmapname = '/lcrc/group/e3sm/data/inputdata/cpl/cpl6/map_r05_to_EC30to60E2r2_smoothed.r150e300.201005.nc'

Differences in namelist 'elm_inparm':
  BASE: co2_type = 'diagnostic'
  COMP: co2_type = 'prognostic'
  BASE: create_crop_landunit = .false.
  COMP: create_crop_landunit = .true.
  BASE: fsurdat = surfdata_0.5x0.5_simyr1850_c190418.nc'
  COMP: fsurdat = surfdata_0.5x0.5_simyr1850_c211019.nc'
  BASE: paramfile = clm_params_c180524.nc'
  COMP: paramfile = clm_params_c211124.nc'
Differences in namelist 'ndepdyn_nml':
  BASE: stream_fldfilename_ndep = fndep_clm_hist_simyr1849-2006_1.9x2.5_c100428.nc'
  COMP: stream_fldfilename_ndep = fndep_elm_cbgc_exp_simyr1849-2101_1.9x2.5_c190103.nc'
  BASE: stream_year_last_ndep = 2005
  COMP: stream_year_last_ndep = 2101
Differences in namelist 'popd_streams':
  BASE: stream_fldfilename_popdens = clmforc.Li_2012_hdm_0.5x0.5_AVHRR_simyr1850-2010_c130401.nc'
  COMP: stream_fldfilename_popdens = elmforc.ssp5_hdm_0.5x0.5_simyr1850-2100_c190109.nc'
  BASE: stream_year_last_popdens = 2010
  COMP: stream_year_last_popdens = 2100
  BASE: flanduse_timeseries = landuse.timeseries_0.5x0.5_hist_simyr1850-2015_c191004.nc'
  COMP: flanduse_timeseries = landuse.timeseries_0.5x0.5_HIST_simyr1850-2015_c211019.nc'
  missing variable: 'check_finidat_year_consistency'
  missing variable: 'check_dynpft_consistency'
  BASE: lnd_rof_coupling_nstep = 1
  COMP: lnd_rof_coupling_nstep = 6
```

**Comment by jenniferholm** [comment]
Ok, thanks for confirming that "nu_com=RD" case wasn't removed. All else makes sense!

**Comment by peterdschwartz** [comment]
Merged to master

**Pull request merged**